### PR TITLE
hardening/894_create_GET_instance_conf_method

### PR DESCRIPTION
**Pull request by pcoello25**
* Partially implements issue #894 
  * Update CHANGES_NEXT_RELEASE neccesary (first time)
* 100% unit tests passed:
```
Results : Tests run: 73, Failures: 0, Errors: 0, Skipped: 0
```
* e2e (unofficial) tests passed:
```
[VALID INSTANCE CONFIGURATION FILE]
$ curl -X GET http://localhost:8081/admin/configuration/instance/usr/cygnus/conf/cygnus_instance.conf
{"success":"true","result" : {"instance":{"CONFIG_FILE":"\/usr\/cygnus\/conf\/agent.conf","AGENT_NAME":"cygnusagent","ADMIN_PORT":"8081","CONFIG_FOLDER":"\/usr\/cygnus\/conf","LOGFILE_NAME":"cygnus.log","CYGNUS_USER":"cygnus","POLLING_INTERVAL":"30"}}

[INEXISTENT FILE NAME]
$ curl -X GET http://localhost:8081/admin/configuration/instance/usr/cygnus/conf/cygnus_instance_inexistent.conf
{"success":"false","result" : { "File not found in the path received" }

[INVALID PATH: Must be `/usr/cygnus/conf` ]
$ curl -X GET http://localhost:8081/admin/configuration/instance/usr/cygnus/configuration/cygnus_instance.conf
{"success":"false","result" : {"Invalid path for a instance configuration file"}

[GETTING A SINGLE PARAMETER]
$ curl -X GET http://localhost:8081/admin/configuration/instance/usr/cygnus/conf/cygnus_instance.conf?param="CONFIG_FILE"
{"success":"true","result" : {"CONFIG_FILE":"\/usr\/cygnus\/conf\/agent.conf"}

[GETTING AN INEXISTENT PARAMETER]
$ curl -X GET http://localhost:8081/admin/configuration/instance/usr/cygnus/conf/cygnus_instance.conf?param="CONFIG_FOLDER_FILE"
{"success":"false","result" : {"Param 'CONFIG_FOLDER_FILE' not found in the instance"}
```
* Assignee: @frbattid

### CODE REVIEW

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java [hunk 1]
```diff
@@ -693,6 +697,80 @@ protected void handleGetAdminConfigurationAgent(HttpServletRequest request, Http
...
+            return;
+        } // if
+        
+        File file = new File(pathToFile);
+                
+        if (file.exists()) {
+            FileInputStream fileInputStream = new FileInputStream(file);
+            Properties properties = new Properties();
+            properties.load(fileInputStream);
+
```
Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: Remove white line

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java [hunk 2]
```diff
@@ -693,6 +697,80 @@ protected void handleGetAdminConfigurationAgent(HttpServletRequest request, Http
...
+            } // if else
+            
+            response.getWriter().println("{\"success\":\"true\","
+                    + "\"result\" : " + jsonObject + "");
+            LOGGER.debug(jsonObject);
+            response.setStatus(HttpServletResponse.SC_OK);
+            
+        } else {
+            response.getWriter().println("{\"success\":\"false\","
+                    + "\"result\" : { \"File not found in the path received\" }");
```
Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: Remove white space after `{`

Comment (pcoello25) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: Yep, after `{` and at the end. Fixed in 54d90b7.

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java (hunk 1 above)
Comment (pcoello25) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/ManagementInterface.java]: Fixed in 54d90b7.

**Comment by frbattid** [comment]
Please, add the related doc :)

**Comment by frbattid** [comment]
LGTM

**Pull request merged**